packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayout.tsx

```diff
@@ -23,7 +23,6 @@ import { LoggedOutSidebar } from '../LoggedOutSidebar';
 
 const Content = styled.div<{ $elevation: Elevation }>`
     display: flex;
-    position: relative;
```

Comment:
I am really not sure about this, but it works.

Basically, there is a two column layout on the welcome screen. And we want to right column with the content that has `position: absolute` to overlap to the left column. 

Before, there was a different left column that was a Trezor logo which we didn't wanted it to be overlayed.
So this might make sense.